ingestion/functions/README.md

```diff
@@ -94,27 +94,53 @@ more generally about Python Lambda development
 The points at which the Lambda integration is most apparent are in testing and
 execution of code.
 
+#### Unit tests
+
 Unit testing is mostly standard `pytest`, with a caveat to be sure that tests
 are run with the correct Python version. E.g.,
 
 ```shell
 python3.6 -m pytest test/my_test.py
 ```
 
-Manual testing/execution uses the SAM CLI. Alongside your function, commit a
-sample JSON input event (see the above documentation), and test the function
-locally by running:
+#### Manual local run
+
+**IMPORTANT**: Local runs still require access to a service account hosted on s3. Follow #754 for updates on how to run functions with your own creds.
+
+Run the stack locally using `/dev/run_stack.sh` and follow the [instructions](https://github.com/globaldothealth/list/blob/main/dev/README.md#permissions) to make sure you're an `admin` to be able to give the role account doing the fetch/parsing the right to call your local stack. This step is described below.
+
+Go to the UI at http://localhost:3002/sources and add a new source for your parser, once you give it a name, a URL and save it, it will be given an ID.
+
+Put that ID in the `retrieval/valid_scheduled_event.json` file.
+
+Next invoke the `RetrievalFunction` like this:
 
 ```shell
 sam build
+SOURCE_API_URL=http://localhost:3001/api sam local invoke "RetrievalFunction" -e retrieval/valid_scheduled_event.json --docker-network=host
```

Comment:
This is the value of `SOURCE_API_URL` in `template.yaml` -- is it somehow overridden, and thus we need to specify it here?

Reply:
ah yeah no, no need for that sorry, removed.